ppr-ui/src/components/parties/composables/useSecuredParty.ts

```diff
@@ -96,21 +97,36 @@ export const useSecuredParty = (props, context) => {
   }
 
   const hasMatchingSecuredParty = (addedParty: PartyIF): boolean => {
-    // store state without newly added party
+    // store state without newly added party.
     const parties = cloneDeep(getAddSecuredPartiesAndDebtors.value.securedParties)
     if (localState.partyType === SecuredPartyTypes.INDIVIDUAL) {
       return parties.some(party =>
-        isEqual(party.personName, addedParty.personName) &&
-        isEqual(party.address, addedParty.address)
+        isObjectEqual(party.personName, addedParty.personName) &&
+        isObjectEqual(party.address, addedParty.address)
```

Comment:
Odd!
`isEqual` docs state it does a proper deep comparison on objects, does it not?

https://lodash.com/docs/4.17.15#isEqual

Reply:
Yea, I would expect it to compare objects as well...

Reply:
Not the way we were using it... The objects being compared were not of the proper type..  I am using isEqual in the isObjectEqual after I normalize...

I kept the normalize logic separate because it can be used for any type of simple object...

Reply:
Fair enough, can't claim that JS is simple when it comes to types and it's (often) forced coercion.